jdi-light/src/main/java/com/epam/jdi/light/asserts/core/FileAssert.java

```diff
@@ -24,19 +25,19 @@
     public static FileAssert assertThatFile(String fileName) {
         return new FileAssert(fileName);
     }
-    private File file;
+    private Safe<File> file = new Safe<>();
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/jdi-testing/jdi-light/pullRequest?prid=6397461)